tests/langchain/test_langchain_databricks_dependency_extraction.py

```diff
@@ -35,6 +35,20 @@ def __init__(
         self.task = task
 
 
+def remove_langchain_modules():
+    langchain_modules = [
+        k
+        for k in sys.modules.keys()
+        if k.startswith("langchain.") or k.startswith("langchain_community.")
+    ]
+    sensitive_modules = {"langchain_community.llms.databricks"}
```

Comment:
Should this be **D**atabricks?
```suggestion
    sensitive_modules = {"langchain_community.llms.Databricks"}
```
Also, it's suspicious that this cannot be removed... I think this is the module that we do want to remove, right?